OpenSim/Framework/Console/CommandConsole.cs

```diff
@@ -717,6 +709,8 @@ public CommandConsole(string defaultPrompt) : base(defaultPrompt)
             Commands.AddCommand(
                 "Help", false, "help", "help [<item>]",
                 "Display help on a particular command or on a list of commands in a category", Help);
+
+            ConsoleMode.DisableQuickEdit();
```

Comment:
As this call is Windows specific, could you wrap it in a conditional against `Util.IsWindows`? See https://github.com/HalcyonGrid/halcyon/blob/a301fc4898985c2b7663f3224e790aed04ccaee4/OpenSim/Framework/Util.cs#L89

Reply:
Will that address the problem with running on Linux? The `kernel32.dll` references don't generate load-time problems? I'd love to make this conditional on Windows, I just didn't think that would work and we needed some form of dynamic loading here to allow Linux to run without the references. I'll make the runtime change for sure, that will either work or at least document the intention.

Reply:
There are several other places in the code that reference `kernel32`-based functions, so no they don't cause runtime or compile time issues assuming the function itself doesn't actually get called.  Hence the runtime check was provided and later refactored into this general purpose static utility property.